src/main/java/com/spotify/confidence/ConfidenceValue.java

```diff
@@ -337,7 +358,7 @@ public com.google.protobuf.Value toProto() {
   public static class List extends ConfidenceValue {
     private final ImmutableList<ConfidenceValue> values;
 
-    public List(java.util.List<ConfidenceValue> values) {
+    private List(java.util.List<ConfidenceValue> values) {
```

Comment:
Users can't instantiate a List with custom `ConfidenceValue`s, and need to use our constrained `of<>` methods above